types/src/validator_set.rs

```diff
@@ -68,6 +70,14 @@ impl ValidatorSet {
     pub fn payload(&self) -> &[ValidatorPublicKeys] {
         &self.0
     }
+
+    pub fn change_event_key() -> EventKey {
```

Comment:
Do you want a way to pass the salt?

Reply:
Not in this case because there is only one `ValidatorSet.Change` event key in the system.